apis/cluster/v1/membercluster_types.go

```diff
@@ -17,6 +17,7 @@ import (
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:JSONPath=`.status.conditions[?(@.type=="Joined")].status`,name="Joined",type=string
 // +kubebuilder:printcolumn:JSONPath=`.metadata.creationTimestamp`,name="Age",type=date
+// +kubebuilder:printcolumn:JSONPath=`.status.agentStatus[?(@.type=="MemberAgent")].lastReceivedHeartbeat`,name="Time-Since-Last-Heartbeat",type=date
```

Comment:
The field name does not indicate the data is from member-agent.

We actually have networking agent running as well.

It may confuse customer.

Reply:
I'm thinking of only showing the heartbeat from member-agent by default.

How about naming as "Seen-Member-Agent-At"?